mongoCollections.ts

```diff
@@ -0,0 +1,18 @@
+import { dbConnection } from "./mongoConnection";
+
+// This will allow you to have one reference to each collection per app
+function getCollectionFn(collection: any) {
```

Comment:
Should this function be exported? or are you planning on only using it to construct mongoCollections?

Reply:
Nope, its used internally for the purpose of constructing the collections in a cleaner way